hardware/raspberrypi/bootmodes/bootflow_2711.md

```diff
@@ -16,6 +16,8 @@ The boot flow for the ROM (first stage) is as follows:-
    * Check SPI EEPROM for second stage loader
       * Success - run second stage bootloader
       * Fail - continue
+* If OTP defines `nRPIBOOT` GPIO and `nRPIBOOT` is low
```

Comment:
This extra text is unecessary. The last step is a loop as described in the previous step. The nRPIBOOT GPIO was already checked at line 11

Reply:
Apologies - I've once again not read the page properly. I will modify the PR so it just fixes the typo.

Reply:
Must be all that Scottish air :scotland: :mountain: :wink:

Reply:
> Must be all that Scottish air

Bit racist Andrew 😋

Reply:
I tried to keep the pseudo code structure as close the ROM implementation as possible, the combination of checking the OTP config + GPIO state does require a strong cup of coffee though